pills/07-working-derivation.xml

```diff
@@ -299,12 +298,15 @@
       clearer.
     </para>
     <para>
-      The value returned by the nixpkgs function is a set. More specifically,
-      it's a set of derivations. Using the <code>with</code> expression we bring
-      them into scope. This is equivalent to the <command>:l &lt;nixpkgs&gt;</command>
-      we used in <application>nix repl</application>; it allows us to easily access derivations
-      such as <varname>bash</varname>, <varname>gcc</varname>, and
-      <varname>coreutils</varname>.
+      The value returned by the nixpkgs function is a set; more specifically,
+      it's a set of derivations. Calling <code>import &lt;nixpkgs> {}</code>
+      into a <function>let</function>-expression creates the local variable
+      <varname>pkgs</varname> and brings it into scope. This has an effect similar to
+      the <command>:l &lt;nixpkgs&gt;</command> we used in <application>nix repl</application>,
```

Comment:
I think the `:l` mention only makes sense with reference to `with` since both bring the attributes inside `pkgs` into the scope.